tests/reports/test_report_views.py

```diff
@@ -50,6 +50,7 @@ def test_authenticated_user_can_post(apiclient, user, comment, admin):
     response = apiclient.post(url, data, format='json')
     assert response.status_code == status.HTTP_201_CREATED
     assert response.data['description'] == 'This comment sucks'
-    assert len(mail.outbox) == 2
-    assert 'A Comment has been reported' in mail.outbox[0].subject
-    assert 'A Comment that you created' in mail.outbox[1].subject
+    assert len(mail.outbox) == 3
+    assert 'A Comment was added to your' in mail.outbox[0].subject
+    assert 'A Comment has been reported' in mail.outbox[1].subject
+    assert 'A Comment that you created' in mail.outbox[2].subject
```

Comment:
But where does it add an comment here? In the factory? If so I would propose to mute signals for the factory. http://factoryboy.readthedocs.io/en/latest/orms.html#disabling-signals